src/components/careers/academy_section.module.css

```diff
@@ -73,6 +80,21 @@
     grid-row-gap: 112px;
   }
 
+  .imgContainer {
+    position: absolute;
+
+    top: -10vh;
+    left: 0;
+
+    width: 100%;
+
```

Comment:
what's the reason for these empty spaces?

Reply:
The linter enforces an empty line between some elements. But I checked now, and I have some that are not required! Will remove these ones right now.